brainscore_vision/benchmarks/geirhos2021/__init__.py

```diff
@@ -0,0 +1,93 @@
+from brainscore_vision import benchmark_registry
+from . import benchmark
+
+DATASETS = ['colour', 'contrast', 'cue-conflict', 'edge',
+            'eidolonI', 'eidolonII', 'eidolonIII',
+            'false-colour', 'high-pass', 'low-pass', 'phase-scrambling', 'power-equalisation',
+            'rotation', 'silhouette', 'sketch', 'stylized', 'uniform-noise']
+
+benchmark_ctr = getattr(benchmark, "Geirhos2021colourAccuracy")
+benchmark_registry['brendel.Geirhos2021colour-top1'] = lambda benchmark_ctr=benchmark_ctr: benchmark_ctr()
```

Comment:
```suggestion
benchmark_registry['brendel.Geirhos2021colour-top1'] = getattr(benchmark, "Geirhos2021colourAccuracy")
```
this should simplify for all